src/components/mols/CouponActivated/index.tsx

```diff
@@ -32,7 +34,12 @@ const CouponActivated: React.FC<CouponProps> = ({
           {couponTitle}
         </h1>
         <div className="flex justify-end mb-1 mt-2 p-0.5 gap-4">
-          <span className="relative text-xl w-32 text-white bg-primary-main flex items-center justify-center px-6 py-3 rounded-full">
+          <span
+            className={cx(
+              getBackgroundCouponColor(Number(couponDiscount)),
```

Comment:
Avoid complicating paramaters.

getColorByDiscountValue(couponDiscount);

inside the util you handle the data. it's something specific to the function itself